src/ray/common/ray_syncer/ray_syncer.h

```diff
@@ -142,6 +142,11 @@ class RaySyncer {
   /// \param message The message to be broadcasted.
   void BroadcastMessage(std::shared_ptr<const RaySyncMessage> message);
 
+  /// Function to force a broadcasting of messages come from a component_id
```

Comment:
Can you write a bit more about the guarantee?

Like if the state is eagerly synced or if there's a delay, and etc.